test/matrices.jl

```diff
@@ -0,0 +1,21 @@
+@testset "Matrices" begin
+  # tests with Aitchson's definitions
+  d = 10
+  D = d+1
+  @test J(d) == ones(d, d)
+  @test F(d) == [I(d) -ones(d)]
+  @test norm(G(D) - I(D)  + J(D)/ D, Inf) < 1e-5
+  @test H(d) == I(d) + J(d)
+
+  square_matrices = [J G H]
+
+  # tests of multiplications
+  for M in square_matrices
+    @test norm(M * [1:d...] - M(d) * [1:d...], Inf) < 1e-5
+    @test norm([1:d...]' * M - [1:d...]' * M(d), Inf) < 1e-5
```

Comment:
Can we rewrite  these tests in terms of equality of approximate equality? `\approx` in Julia.

Reply:
done